docs/azure/nRFCloudLocationServices.rst

```diff
@@ -42,3 +42,42 @@ Store the service key into the key vault as follows:
    az keyvault secret set --vault-name ${APP_NAME:-nrfassettracker} \\
      --name nrfCloudCellLocationServiceKey \\
      --file *location of your Cell Location Service Key file*
+
+Assisted GPS Location Service
+*****************************
+
+You can enable your devices to request assisted GPS data using `nRF Cloud's Assisted GPS Location Service API <https://api.nrfcloud.com/v1#tag/Assisted-GPS>`_ in your deployment.
+
+To use the API, set the ``enableNrfCloudAGPSLocationService`` parameter to ``true`` and set the ``nrfCloudTeamId`` parameter to your team ID when deploying the solution.
+
+.. parsed-literal::
+   :class: highlight
+
+    az deployment group create \\
+        --resource-group ${RESOURCE_GROUP:-nrfassettracker} \\
+        --mode Complete \\
+        --name enable-unwiredlabs \\
+        --template-file azuredeploy.json \\
+        --parameters \\
+            appName=${APP_NAME:-nrfassettracker} \\
+            appRegistrationClientId=$APP_REG_CLIENT_ID \\
+            b2cTenant=${B2C_TENANT:-nrfassettrackerusers} \\
+            keyVaultName=${APP_NAME:-nrfassettracker} \\
+            enableNrfCloudAGPSLocationService=true \\
+            nrfCloudTeamId=*your team ID*
+
+This command enables the ``agpsQueuedDeviceRequestsHandler`` function to resolve A-GPS requests from devices using the nRF Cloud Assisted GPS Location Service API.
+
+Store the service key into the key vault as follows:
+
+.. parsed-literal::
+   :class: highlight
+
+   # Grant the current user set permission to the key vault secrets
+   USER_OBJECT_ID=`az ad signed-in-user show --query objectId -o tsv`
+   az keyvault set-policy --name ${keyVaultName} --object-id ${USER_OBJECT_ID} --secret-permissions set
+
+   # Store the API key
+   az keyvault secret set --vault-name ${APP_NAME:-nrfassettracker} \\
+     --name nrfCloudAGPSLocationServiceKey \\
+     --file *location of your Assisted GPS Location Service Key file*
```

Comment:
Maybe A-GPS to make this a bit shorter.

Reply:
But this is the name of the service: 
![image](https://user-images.githubusercontent.com/188915/135445643-b39c9427-f358-4108-8de5-7b2a6d030b1b.png)